documentation/postgres-faq.md

```diff
@@ -0,0 +1,51 @@
+# Postgres FAQ
+
+FAQ for anything postgres related
+
+## Admin password change
+
+The Postgres admin user and password is created by the postgres terraform module.
+
+https://github.com/DFE-Digital/terraform-modules/tree/main/aks/postgres
+
+If you need to change the password for any reason, then use the following procedure which will trigger a new password on the next deployment.
+Note that make commands and directories may be slightly different for your service, so check within your Makefile before running.
+
+This example changes the postgres admin password for the development env of the service
+
+1. Set terraform env
+```
+$ make development terraform-init
+```
+
+2. Change to the terraform directory (may be different for your service)
+```
+$ cd terraform\aks
```

Comment:
We could use the `terraform -chdir terraform/aks ...` option. It avoids chaning directory back and forth